bitnami/thanos/templates/query/deployment.yaml

```diff
@@ -86,21 +86,32 @@ spec:
             - --store={{ . }}
             {{- end }}
             {{- if $query.grpcTLS.server.secure }}
+            {{- if $query.grpcTLS.server.existingSecret }}
+            - --grpc-server-tls-cert=/tls/server/tls.crt
+            - --grpc-server-tls-key=/tls/server/tls.key
+            - --grpc-server-tls-client-ca=/tls/server/ca.crt
```

Comment:
Thanks. I would suggest keeping the same names as in the secret that the chart creates (`cert.pem`, `key.pem`, `ca.pem`) so that we can avoid all this if-else logic. What do you think?

Reply:
@alemorcuq a case that I have for this is that I'm using certificates generated by cert-manager, which have keys similar to the one I provided. I agree though that these are not consistent with the `.pem`, which is already being used by the template. 

I would suggest giving the user the flexibility to specify the key names, similar to what I had in my first commit. The default keys can be the ones mentioned  (`cert.pem`, `key.pem`, `ca.pem`), but the user can modify them in case of relying on certificates generated by cert-manager

Reply:
Agree with to permit the users to specify the keys but respecting the default one.

Reply:
@alemorcuq should I revert back to my first commit?

Reply:
I guess we could use something like this https://github.com/bitnami/charts/pull/5189/files

Reply:
I like the idea, would look something like this

```yaml
existingSecret:
  name: ''
  keyMapping:
    ca-cert: ca.pem # <-- default
    cert: cert.pem # <-- default
    key: key.pem # <-- default 
     
```
     
@alemorcuq @dani8art should I continue on it to reflect the snippet above?

Reply:
Any updates on this? should I proceed with the above snippet?

Reply:
I like the idea! Let's start working on the PR. @alemorcuq will review it later ;)

Thank you so much!

Reply:
Yes, @IbraheemAlSaady, thank you very much. Sorry for the late response. @dani8art will take care of reviewing your PR.